skribenten-web/frontend/src/routes/saksnummer_/$saksId/brevvelger/route.tsx

```diff
@@ -142,7 +209,29 @@ function Brevmaler({ letterTemplates }: { letterTemplates: LetterMetadata[] }) {
                   `}
                 >
                   {brevmalerGroupedByType[type].map((template) => (
-                    <BrevmalButton key={template.id} letterMetadata={template} />
+                    <BrevmalButton
```

Comment:
Her bør vi sjekke om det finnes en kladd for denne malen, og velge om vi skal rute til `kladd/$brevId` eller `$templateId`

Reply:
🤷 spørsmålet blir kanskje hva vi skal gjøre hvis vi allerede har en kladd for et brev?

Reply:
Ja, la det være sånn som dette. Det var jeg som husket et tidligere design, fra før den "kladder" kategorien kom.